pyproject.toml

```
@@ -0,0 +1,49 @@
+[build-system]
+requires = ["setuptools"]
```

```suggestion
requires = ["setuptools", "wheel"]
```

If you add the `wheel` directive here, a pure Python wheel will also be generated during the build phase. This wheel can be useful for compatibility with some Python environments, e.g., [Pyodide](https://pyodide.org/en/stable/development/new-packages.html#determining-if-creating-a-pyodide-package-is-necessary)